static/app/components/pullRequestLink.tsx

```diff
@@ -16,9 +16,9 @@ function renderIcon(repo: Repository) {
 
   switch (providerId) {
     case 'github':
-      return <IconGithub size="xs" />;
+      return <IconGithub size="xs" data-test-id="pull-request-github" />;
```

Comment:
we will have to improve the accessibility of our svgs afterward, so we will be able to query "aria-labelledby"